emails/Makefile

```diff
@@ -6,7 +6,7 @@ clean:
 
 build-mjml:
 	npx mjml \
-		--config.beautify true \
+		--config.beautify false \
```

Comment:
We have to turn this off because the beautifier messes with the golang syntax in the CSS file.